src/services/auth.service.js

```diff
@@ -79,3 +84,34 @@ function register(name, email, phone, username, password1, password2) {
             }
         });
 }
+function chechUser(userId) {
+    console.log(userId);
+     return axios.get(
+        stringFormat(`${config.commonConfig.$apiUrl}/${config.userConfig.api.checkUser.endpoint}`, userId)
+    )
+
+        .then((response) =>{
+            console.log('response =', response.data);
```

Comment:
1. Check if response has `username` block.
YES - return that.
NO - throw error. (catch it in store and dispatch `alertStore/error`)

Reply:
No need to `console.log` unless not testing out. Production JS environment does not allow `console.log`